### PR TITLE
Fix incorrect selection of FASTA hits.

### PR DESCRIPTION
**Pull request by vivekiitkgp**
Signed-off-by: Vivek Rai <vivekraiiitkgp@gmail.com>

### CODE REVIEW

**Comment by yeban** [comment]
I think this won't highlight corresponding '.hitn' if selected from sequence viewer (modal).

**Comment by yannickwurm** [comment]
It seems having to synchronise [ ] select buttons makes everything complicated. For simplicity's sake & fewer test scenarios I suggest removing the [ ]select button from the viewing modal for the time being.

**Comment by vivekiitkgp** [comment]
@yeban It seems to be resolved now.

File: views/result.erb [hunk 1]
```diff
@@ -109,7 +109,7 @@
                   <div class="hit-links text-right">
                     <label>
                       <input
-                      type="checkbox" value=<%="#{ hit.accession }"%>>
+                      type="checkbox" value="<%="Query_#{query.number}_hit_#{hit.number}"%>">
```
Comment (yeban) [inline views/result.erb]: Why not set the value directly to '#Query_1_hit_1'? Like BS does it for `data-target`?

Comment (vivekiitkgp) [inline views/result.erb]: Depends on how one interprets the difference between `data-target` and
`value`. While former gives a sense that we are referring to a div or any
DOM element, latter doesn't.

So, if we are directly using `#Query_1_hit_1`, then might as well use
`data-target`.

Comment (yeban) [inline views/result.erb]: Let's use `data-target` coz value is needed for generating the link.

**Comment by yeban** [comment]
via 46edb6d4641dd4f56

**Pull request closed without merging**